models/issues/issue.go

```diff
@@ -1449,6 +1462,11 @@ func issuePullAccessibleRepoCond(repoIDstr string, userID int64, org *organizati
 }
 
 func applyAssigneeCondition(sess *xorm.Session, assigneeID int64) *xorm.Session {
+	if assigneeID == -1 {
+		return sess.Join("LEFT", "issue_assignees", "issue.id = issue_assignees.issue_id").
```

Comment:
Use `NotExist` is better.

Reply:
could you provide an example - never saw an example of `builder.NotExists()`

Reply:
> could you provide an example - never saw an example of `builder.NotExists()`

I have done that in the previous comment.